### PR TITLE
Paice's method of evaluating stemming algorithms.

### PR DESCRIPTION
**Pull request by lade**
Paice's method of evaluating stemming algorithms by calculating understemming and overstemming indexes, stemming weight, and error-rate relative to truncation (ERRT).

Based on Paice: An evaluation method for stemming algorithms.

### CODE REVIEW

File: nltk/metrics/paice.py [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+
+What is required:
+ - A dictionary of words grouped by their real lemmas
+ - A dictionary of words grouped by stems from a stemming algorithm
+
+When these are given, Understemming Index (UI), Overstemming Index (OI),
+Stemming Weight (SW) and Error-rate relative to truncation (ERRT) are counted.
+
+References:
+Chris D. Paice: An Evaluation Method for Stemming Algorithms.
```
Comment (jnothman) [inline nltk/metrics/paice.py]: Please provide a full citation such as:

```
Chris D. Paice (1994). An evaluation method for stemming algorithms. In Proceedings of SIGIR, 42--50.
```

Comment (lade) [inline nltk/metrics/paice.py]: Thanks for correcting; that's the correct article.

**Comment by stevenbird** [comment]
Thanks @lade.

**Comment by stevenbird** [comment]
@lade note that I will add the standard NLTK copyright statement. https://github.com/nltk/nltk/wiki/Developers-Guide

**Comment by lade** [comment]
Sure!

**Pull request merged**